### ISSUE TITLE
Default executor is not using the expected node version

### ISSUE THREAD

**Issue by Maxwell2022** [CircleCI-Public/serverless-framework-orb#20]
**Orb Version: v1.0.1**

**Describe the bug**

[`Serverless 2.69.0`](https://github.com/serverless/serverless/releases/tag/v2.69.0) is requiring Node 16 binaries.
The default executor version is using python `3.8-node` which embeds node 14.

- Deployment fails to upload the artifacts to S3

This is the error message you can expect
> Resource handler returned message: "Uploaded file must be a non-empty zip (Service: Lambda, Status Code: 400, ...)"

**To Reproduce**

run deploy with default executor version

**Expected behavior**

deployment succeed

**Additional context**

N/A

**Work around**

Set the `VERSION` environment alongside the executor to set the SLS version to 2.68.x

**Possible fix**
- Allow parameter to chose which version of SLS to install and then use `curl -o- -L https://slss.io/install | VERSION=2.21.1 bash`
- Update default python image version

**Comment by KyleTryon** [comment]
```
docker run -it cimg/python:3.8-node bash
...
circleci@6c47c8edc792:~/project$ node --version
v16.14.2
```

3.10 is the latest version of python available which appears to have the same Node version present. The version parameter has been added and will be available in the next release (due to come out today or tomorrow). Please keep an eye out on the releases/tags for a new version coming shortly.

**Comment by KyleTryon** [comment]
Hello,
Thank you all for reporting this issue. We have just released version 2.0.0 which should resolve this problem. Please try the latest version of the orb and let us know if you experience any issues.

https://github.com/CircleCI-Public/serverless-framework-orb/releases/tag/v2.0.0
https://circleci.com/developer/orbs/orb/circleci/serverless-framework
